src/RoslynAnalyzers/Tools/GenerateDocumentationAndConfigFiles/CodeFixerExtensions.cs

```diff
@@ -52,14 +52,16 @@ public static ImmutableArray<CodeFixProvider> GetFixers(this AnalyzerFileReferen
                                 }
                             }
                         }
-                        catch
+                        catch (Exception ex)
                         {
+                            Console.WriteLine($"Error creating instance of {typeInfo.FullName}: {ex.Message}\r\n{ex}");
                         }
                     }
                 }
             }
-            catch
+            catch (Exception ex)
             {
+                Console.WriteLine($"Error processing analyzer file reference: {ex.Message}\r\n{ex}");
```

Comment:
If we keep this, it would be useful to include `Path.GetFileName(analyzerFileReference.FullPath)` in the message.

Reply:
sgtm.